src/content/en/updates/2017/09/autoplay-policy-changes.md

```diff
@@ -208,7 +219,7 @@ Key Point: An <code>AudioContext</code> must be created or resumed after the
 document received a user gesture to enable audio playback.
```

Comment:
I think this point should really change to "If an AudioContext is created prior to the document receiving a user gesture, it will be created in the "suspended" state, and you will need to call resume() after a user gesture is received."

Reply:
Done.